src/server/routes/users.js

```diff
@@ -135,47 +138,98 @@ router.post('/create', adminAuthMiddleware('create a user.'), async (req, res) =
  * Route for updating user role
  */
 router.post('/edit', adminAuthMiddleware('update a user role'), async (req, res) => {
+	
 	const validParams = {
 		type: 'object',
-		required: ['users'],
+		required: ['user'],
 		properties: {
-			users: {
-				type: 'array',
-				items: {
-					type: 'object',
-					required: ['email', 'role'],
-					properties: {
-						email: {
-							type: 'string'
-						},
-						role: {
-							type: 'string',
-							enum: Object.values(User.role)
-						}
-
+			user: {
+				type: 'object',
+				required: ['id', 'email', 'role'],
+				properties: {
+					id: {
+						type: 'integer'
+					},
+					email: {
+						type: 'string'
+					},
+					role: {
+						type: 'string',
+						enum: Object.values(User.role)
+					},
+					password: {
+						type: 'string'
+					},
+					note: {
+						type: 'string'
 					}
 				}
 			}
 		}
 	};
+
 	if (!validate(req.body, validParams).valid) {
 		res.status(400).json({ message: 'Invalid params' });
 	} else {
+
```

Comment:
OED does prefer not to have blank lines in places such as this.

Reply:
Got it.